Feature/variables

martakoss27 opened the pull request.

krystianklimczak reviewed (commented):
please change names of variables without using colors, instead of it try to implement the name of what kind of color is using it to.
for example try to use these : primary, secondary, light, dark.
because if we gonna change one of those color for example our white gonna be grey the name will be not correct
I hope you understand my idea
implement font size variables 
implement font weight variables
implement colors of hover or focus add some alpha color to change them
change and request again

martakoss27 commented:
I changed the names, added font sizes, font weights, hover and focus colors and some additional vars. If there's anything else I should change let me know.

annienf17 commented:
Hi Marta, 
Many thanks for your input.
Following the convention S/M/L small/medium/large, would be perfect to change variable $font-size-big: 30px; to $font-size-large: 30px;

Instead of using #FFFFFF the better option is to use rgba(255, 255, 255, 1) or rgb(255, 255, 255) where rgba will allow us to manipulate opacity.

krystianklimczak commented:
As Annie suggested we should use convention S/M/L.
Good idea with using rgba color it's gonna be more easy
From me one suggestion, try to not using numbers in vars, insted of it you can write "primary" "secondary" "main" "aside" by the way start naming variable with general word at the beggining : $text/background/font instead of using "additional" first, you can paste it at the end. So for example new variable will be : $text-color-additional or $text-color-main etc.

krystianklimczak commented on src/sass/partials/_variables.scss:
```diff
@@ -3,10 +3,10 @@ $font-weight-medium: 500;
-$additional-text-color-2: #545454;
+$text-color-light: rgb(255, 255, 255);
+$text-color-dark: rgb(0, 0, 0);
+$additional-text-color-primary: rgb(255, 107, 8);
+$additional-text-color-secondary: rgb(84, 84, 84);
```
keep your idea $text-color-additional-primary/secondary start variable with general word

krystianklimczak reviewed (commented):
usefull article : https://www.fixate.it/blog/make-your-css-variable-names-suck-less
It would help you understand

krystianklimczak reviewed (approved):
Annie check if you agree with all and we gonna merge it. ✌️

annienf17 commented:
yes, everything is fine, go on, mamy thanks!
